packages/fiori/src/WizardTab.js

```diff
@@ -284,6 +278,17 @@ class WizardTab extends UI5Element {
 	get hasTexts() {
 		return this.heading || this.subheading;
 	}
+
+	get accInfo() {
+		return {
+			"role": this._wizardTabAccInfo.role ? this._wizardTabAccInfo.role : this.role,
+			"ariaSetsize": this._wizardTabAccInfo.ariaSetsize ? this._wizardTabAccInfo.ariaSetsize : this.ariaSetsize,
+			"ariaPosinset": this._wizardTabAccInfo.ariaPosinset ? this._wizardTabAccInfo.ariaPosinset : this.ariaPosinset,
+			"ariaLabel": this._wizardTabAccInfo.ariaLabel ? this._wizardTabAccInfo.ariaLabel : this.ariaLabel,
+			"ariaCurrent": this._wizardTabAccInfo.ariaCurrent ? this._wizardTabAccInfo.ariaCurrent : this.ariaCurrent,
```

Comment:
do we need to pass ariaCurrent and ariaDisabled as part of the accInfo as they are correctly calculated via this.selected and this.disabled?

Reply:
if this is the case could you change the ariaCurrent getter to return true if the step is selected?